abci/client/local_client.go

```diff
@@ -26,15 +27,15 @@ var _ Client = (*localClient)(nil)
 // methods of the given app.
 //
 // Both Async and Sync methods ignore the given context.Context parameter.
-func NewLocalClient(mtx *sync.Mutex, app types.Application) Client {
+func NewLocalClient(logger log.Logger, mtx *sync.Mutex, app types.Application) Client {
 	if mtx == nil {
 		mtx = new(sync.Mutex)
```

Comment:
Not related to your change, but yowzers. 🤨

Reply:
lol, yes.